compass/run.py

```diff
@@ -1,10 +1,140 @@
 import argparse
 import sys
 import os
+import pickle
+import configparser
+import time
+import numpy
 
-from compass.suite import run_suite
-from compass.testcase import run_test_case
-from compass.step import run_step
+from mpas_tools.logging import LoggingContext
+
+
+def run_suite(suite_name):
+    """
+    Run the given test suite
+
+    Parameters
+    ----------
+    suite_name : str
+        The name of the test suite
+    """
+    with open('{}.pickle'.format(suite_name), 'rb') as handle:
```

Comment:
Should there be an exception here to gracefully error if the specified suite's pickle files does not exist?  I realize that it's nor practical to code exceptions for every possible user error, but this one would be pretty high level from a user perspective.

Reply:
Yes, that's a good suggestion.